### PR TITLE
#issue 25  Resolved Page breaking down at bottom

### PR DESCRIPTION
**Pull request by pranjalagrawal2012**
## Related Issue 

Back Arrow in each page was not working properly 

Breaking body page on bottom

Closes: #[issue number that will be closed through this PR]

issue-23

issue-25

### Describe the changes you've made

Went through various upgradation of packages and flutter updating, Check every page for the back arrow issue 

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Mention any unusual behaviour of your code (Write NA if not)
Any unusual behaviour of your code

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Additional Info (optional)
Any additional information you want to give

### CODE REVIEW

**Comment by hiashutoshsingh** [comment]
WhatsApp Image 2021-03-16 at 10 48 59 AM (1)
WhatsApp Image 2021-03-16 at 10 48 59 AM
WhatsApp Image 2021-03-16 at 10 49 00 AM

@pranjalagrawal2012 UI is breaking on some phone. I tested this on a 5.5-inch android phone.
What you can do here to make it responsive on all phones. This is a very easy-peasy task.

Let me know if you need some help with this.

**Comment by hiashutoshsingh** [comment]
It's not recommended to send a lock file(e.g pubspec.lock) from another environment. It should remain intact with one user only. @pranjalagrawal2012

**Review by hiashutoshsingh — CHANGES_REQUESTED** [review]
1) Change the pull request name to something meaningful.
2) Fix the UI changes 

@pranjalagrawal2012

**Comment by pranjalagrawal2012** [comment]
Made the Changes and resolved the issue of screen overflowing @hiashutoshsingh @shagun25

**Comment by hiashutoshsingh** [comment]
@pranjalagrawal2012 Hey, you look into these conflicts. I hope you are behind master. Please do this carefully this may break something.

File: Sheheroes_WatchOS/pubspec.lock [hunk 1]
```diff
@@ -63,7 +63,7 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
```
Comment (hiashutoshsingh) [inline Sheheroes_WatchOS/pubspec.lock:64]: remove this file from commit, lock file is not added usually.

File: lib/pages/login_page.dart [hunk 2]
```diff
@@ -445,7 +447,7 @@ class _LoginPageState extends State<LoginPage>
 
   Widget _buildSignUp(BuildContext context) {
     return Container(
-      padding: EdgeInsets.only(top: 23.0),
+      padding: EdgeInsets.only(top: 0.0),
```
Comment (hiashutoshsingh) [inline lib/pages/login_page.dart:450]: if no padding is need then remove this padding field rather that assigning its value to 0

File: pubspec.lock [hunk 3]
```diff
@@ -63,7 +63,7 @@ packages:
       name: async
```
Comment (hiashutoshsingh) [inline pubspec.lock:63]: remove this file also from commit.

**Review by hiashutoshsingh — CHANGES_REQUESTED** [review]
Please resolve these changes.

**Comment by hiashutoshsingh** [comment]
Hey, @pranjalagrawal2012  didn't get an update from your side.
Is there any issue are facing to making changes. Let me know.

File: Sheheroes_WatchOS/pubspec.lock (hunk 1 above)
Comment (pranjalagrawal2012) [reply Sheheroes_WatchOS/pubspec.lock:64]: okay @hiashutoshsingh

File: lib/pages/login_page.dart (hunk 2 above)
Comment (pranjalagrawal2012) [reply lib/pages/login_page.dart:450]: okay @hiashutoshsingh

File: pubspec.lock (hunk 3 above)
Comment (pranjalagrawal2012) [reply pubspec.lock:63]: okay @hiashutoshsingh

File: Sheheroes_WatchOS/pubspec.lock (hunk 1 above)
Comment (pranjalagrawal2012) [reply Sheheroes_WatchOS/pubspec.lock:64]: I have removed the padding field whose value is assigned to 0 . Can you guide me how to remove the file from commit ??

File: pubspec.lock (hunk 3 above)
Comment (pranjalagrawal2012) [reply pubspec.lock:63]: @hiashutoshsingh  Can you guide me how to remove the file from commit ??

File: lib/pages/login_page.dart (hunk 2 above)
Comment (pranjalagrawal2012) [reply lib/pages/login_page.dart]: @hiashutoshsingh  have removed the padding field whose value is assigned to 0

File: Sheheroes_WatchOS/pubspec.lock (hunk 1 above)
Comment (pranjalagrawal2012) [reply Sheheroes_WatchOS/pubspec.lock]: @hiashutoshsingh  Can you please review the same ?

File: pubspec.lock (hunk 3 above)
Comment (hiashutoshsingh) [reply pubspec.lock:63]: make a copy of the file at another place and delete that file then commit it.

**Comment by hiashutoshsingh** [comment]
Hey, closing this PR because it is awaiting for a long time. Feel free to open again with new changes.

Thanks.

**Pull request closed without merging**